### DISCUSSION TITLE
Exiobase import factors report excludes some sectors that do not have IEFs?

### DISCUSSION THREAD

**Discussion by cclin130** [USEPA/USEEIO#43] [Q&A] [answered]
The import factors report lists the following sectors as those without IEFs because no equivalent commodities were found in Exiobase:
<img width="723" alt="image" src="https://github.com/user-attachments/assets/50698471-b7d9-47b3-8f6e-ab65439fbad9">

But when I download import factors off of the [data.gov website](https://catalog.data.gov/dataset/useeio-models-with-import-emission-factors-for-greenhouse-gases-for-2017-2022-from-exiobas), the file I download is missing other summary-level sectors
<img width="1192" alt="image" src="https://github.com/user-attachments/assets/6201a846-fc3f-4ed3-9f7a-f6d59e0e3b63">

For example, summary sector `23` is not present

**Comment by bl-young** [comment, marked as answer]
I believe that Table 4 is for sectors WITH imports, but no IEF. I don't believe there are any imports of `23` in the IO tables.

**Reply by cclin130** [reply]
Yep, I noticed that `23` is not in the BEA/Census import shares data. Thank you!

**Comment by cclin130** [comment]
@bl-young I am seeing that the Exiobase IEFs _do_ have the `Used` summary-level sector. Am I misinterpreting the Table 4 in another way?

<img width="940" alt="image" src="https://github.com/user-attachments/assets/5c4c744b-14d2-4022-bab1-0d98a2bcb812">

**Reply by bl-young** [reply]
Yes, interesting. I recall [fixing an error](https://github.com/USEPA/USEEIO/issues/23#issuecomment-1960050592) to make sure that was added. So why is it in Table 4? I will have to look...

**Reply by cclin130** [reply]
Relatedly I cannot find  `621 Ambulatory Health Care Services`, `622`, `623` or `Other` in the import shares file -- I am not sure if there is a mistake in my import shares file (or if I'm looking at the wrong file). Is this expected?

**Reply by bl-young** [reply]
I believe the assessment of whether imports occur is done in the IO tables themselves, not based on the BEA/Census imports data. So this seems like a case where the IO tables may have data in the imports vector for these sectors, but perhaps the actual trade data doesn't directly support that (based on our mappings - these would come from the BEA services dataset, I believe). This is just my immediate reaciton.

**Reply by cclin130** [reply]
Understood. I checked for `621` and confirmed that within Exiobase, there are non-zero bilateral trade values in `Health and social work services (85)` to the US.

It feels a bit confusing to me in the report to write:

> IEFs were generated for all USEEIO commodities that have imports except for those listed in Table 4. No equivalent commodities were found in EXIOBASE for these commodities listed.

Because the `621, 622, 623` sectors have entries in the sector concordance files -- so EXIOBASE _does_ have these commodities. But the actual BEA trade data does not support the fact that Exiobase says these sectors are imported, and therefore they are dropped. `Used` and `Other` are not present in Exiobase and align with what's mentioned in the report (and I think this answers our confusion regarding `Used`).

Within CEDA, I have started the same exercise and will make note of which sectors (1) are not in CEDA at all (this is `Used` and `Other`), (2) are in CEDA but have no USA trade in CEDA

**Reply by bl-young** [reply]
I'm having a hard time re-creating Table 4 so I'm a bit confused exactly what led to that list. There are a few other sectors that show imports in the IO data but do not appear in the BEA/Census imports data and so do not have import factors (e.g. `713`) - I'm not sure why they didn't make it into Table 4.

> so EXIOBASE does have these commodities. But the actual BEA trade data does not support the fact that Exiobase says these sectors are imported, and therefore they are dropped.

^ This is correct. The text in the report is not quite right then.

**Reply by bl-young** [reply]
Ok it turns out that one issue is that I believe the values in that table are from 2017, not 2019. In 2017 there are no imports for `713`, so that was actually correct.

There are two separate issues we were trying to document here, which I think got conflated and poorly explained. One is where we don't have good mappings (`Other` is the prime example here). A second issue is a mismatch of imports data; where we don't have imports data from BEA and Census we can not create an aggregate import factor as there is no way to aggregate across regions. This is the case for 621, 622, and 623. Also the final issue is that this table was created before we implemented a fix and so `Used` shouldn't be in there at all.

So yes, your thought here makes sense.
> Within CEDA, I have started the same exercise and will make note of which sectors (1) are not in CEDA at all (this is Used and Other), (2) are in CEDA but have no USA trade in CEDA

**Reply by cclin130** [reply]
sounds good, I appreciate you looking into this! I will add a note about sectors without import EFs in CEDA in our report draft
